libexec/rtld-elf/rtld_c18n.c

```diff
@@ -245,7 +245,7 @@ tramp_should_include(const Obj_Entry *reqobj, const struct tramp_data *data)
  * Set a dummy Restricted stack so that trampolines do not need to test if the
  * Restricted stack is valid.
  */
-static void *dummy_stack = &dummy_stack;
+static void *dummy_stack[1] = { &dummy_stack[0] };
```

Comment:
Why? T and T[1] are both a single object.

Reply:
Just to make the intent clearer that this is a `stack` whose bottom element points to itself.